narwhal/primary/src/state_handler.rs

```diff
@@ -447,6 +437,57 @@ impl RandomnessState {
                 return;
             }
         };
+
+        // Try to verify the aggregated signature all at once. (Should work in the happy path.)
+        if ThresholdBls12381MinSig::verify(dkg_output.vss_pk.c0(), &round.signature_message(), &sig)
```

Comment:
`round` seems to be set to `self.randomness_round` by the only caller to this function, could there be a case they differ?

I want to make sure that we are verifying the right message here

Reply:
there are two callers, the self-call sets it to `self.randomness_round` and the call triggered by reciving partial sigs externally sets it to the round indicated in that RPC (see towards end of file)